Makefile

```diff
@@ -68,8 +68,13 @@ core-functional-test-coverage:
 core-javadoc:
 		(./code/gradlew -p code/android-core-library dokkaJavadoc)
 
-core-publish:
-		(./code/gradlew -p code/android-core-library  publishReleasePublicationToSonatypeRepository)
+core-publish-staging: clean
+		(./code/gradlew -p code/android-core-library assemblePhoneRelease)
```

Comment:
Can you depend on the make target for core-assemble-release instead? 
Similar changes for all other make targets

Reply:
Changed this for the targets touched with this PR. Will change other targets in a separate pr to limit diff.